### PR TITLE
Addidng the drone_console alias as well as the mavlinkbridge service …

### PR DESCRIPTION
**Pull request by 25cent9**
…for future on board computers.

Plz **_LGTM_**

### CODE REVIEW

File: config/alias.txt [hunk 1]
```diff
@@ -0,0 +1,2 @@
+alias drone_console="sudo service mavlinkbridge stop && sudo mavproxy.py --master=/dev/serial/by-id/usb-3D_Robotics_PX4_FMU_v2.x_0-if00"
```
Comment (ClayMav) [inline config/alias.txt]: Can you instead make this file into a `dronerc` file

**Review by ClayMav — CHANGES_REQUESTED** [review]
Can you make a readme for `config/` with explanations for the purpose and use of each thing in here, as well as directions to use each. For the `dronerc` file the directions should be to use `source dronerc` to apply the alias

**Comment by ClayMav** [comment]
Sorry tiny commit took 400 years to get reviewed, I forgot it was here.

File: config/alias.txt (hunk 1 above)
Comment (25cent9) [reply config/alias.txt]: Done

**Comment by 25cent9** [comment]
Made changes accordingly.

**Review by ClayMav — APPROVED** [review]
👍

**Pull request merged**